README.md

```diff
@@ -1,2 +1,103 @@
 # Scalafix rules for Typelevel projects
 
+[![Continuous Integration](https://github.com/typelevel/typelevel-scalafix/actions/workflows/ci.yml/badge.svg)](https://github.com/typelevel/typelevel-scalafix/actions/workflows/ci.yml)
+[![License](https://img.shields.io/github/license/typelevel/typelevel-scalafix.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Discord](https://img.shields.io/discord/632277896739946517.svg?label=&logo=discord&logoColor=ffffff&color=404244&labelColor=6A7EC2)](https://discord.gg/D7wY3aH7BQ)
+
+This is a set of Scalafix rules to apply automated rewrites and linting in [Typelevel](https://github.com/typelevel) projects.
```

Comment:
Well, really outside of Typelevel too, right? Any project with Typelevel deps :)

Reply:
😆 yeah good point, not sure why I worded it that way